Base/splitter.h

```diff
@@ -0,0 +1,42 @@
+#ifndef SPLITTER_H__
+#define SPLITTER_H__
+
+struct SplitterRecord
```

Comment:
Shouldn't this kind of configuration structure be attached to a specific `Splitter` implementation?